recipes-ni/initscripts-nilrt/files/mountconfig

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+# Copyright (c) 2012-2013 National Instruments.
+# All rights reserved.
+
+partition_label="boot-config"
+volume_label="config"
+volume_number="3"
+mount_point="/etc/natinst/share"
+mount_args="-o sync"
+RESTORECON="/sbin/restorecon"
+
+if [ -e /etc/init.d/mountutils ] ; then
+	# include common utilities
+	. /etc/init.d/mountutils
+fi
+
+# Bail out on first error
+set -o errexit
+
+mount_config()
+{
+	# Don't mount if already mounted
+	if ! grep -q " $mount_point " /proc/self/mounts; then
+		if cat /sys/class/mtd/mtd*/name 2>/dev/null | grep -q ^$partition_label\$
+		then
+			# If an mtd partition has this partition label, use
```

Comment:
The entries that refer to mtd and mount_ubi_volume are Zynq-specific and can be removed. The mount command on line 33 is enough. Do you think it's a good idea to edit these files as you move them or should it be done in the next update? Can you create a task for it or something if you pick the latter option?

Reply:
Removed the file altogether as remaining changes aren't necessary. I can edit them as I go.